### PR TITLE
Cloud Console Metrics Page

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-6983

Add the serverless Metrics page for the cloud console.
There is significant overlap with the UI SQL Dashboard, but there is no directly corresponding page in the DB Console, so this page does not use the same include structure as other Cloud observability pages.

### CODE REVIEW

**Review by THardy98 — APPROVED** [review]
LGTM
i think @ericharmeling is also taking a look through before you merge

File: cockroachcloud/metrics-page.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+### Transaction Restarts
+
+This graph shows the number of transactions restarted across the cluster.
+
+See the [Transaction Retry Error Reference](../{{site.versions["stable"]}}/transaction-retry-error-reference.html) for details on the errors that caused the transaction to restart.
+
+### SQL Statement Errors
+
+This graph shows the 10-second average of the number of SQL statements that returned a [planning](../{{site.versions["stable"]}}/architecture/sql-layer.html#sql-parser-planner-executor), [runtime](../{{site.versions["stable"]}}/architecture/sql-layer.html#sql-parser-planner-executor), or [retry error](../{{site.versions["stable"]}}/transactions.html#error-handling) across all nodes.
```
Comment (ericharmeling) [inline cockroachcloud/metrics-page.md:63]:
```suggestion
This graph shows a moving average of the number of SQL statements that returned a [planning](../{{site.versions["stable"]}}/architecture/sql-layer.html#sql-parser-planner-executor), [runtime](../{{site.versions["stable"]}}/architecture/sql-layer.html#sql-parser-planner-executor), or [retry error](../{{site.versions["stable"]}}/transactions.html#error-handling) across all nodes.
```

File: cockroachcloud/metrics-page.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+See the [Statements page](statements-page.html) for more details on the cluster's SQL statements.
+
+### SQL Statement Full Scans 
+
+This graph shows the total number of full table and index scans across all nodes in the cluster.
```
Comment (ericharmeling) [inline cockroachcloud/metrics-page.md:69]:
```suggestion
This graph shows a moving average of the number of statements with full table and index scans across all nodes in the cluster.
```

File: cockroachcloud/metrics-page.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+[Examine the statements](../{{site.versions["stable"]}}/sql-tuning-with-explain.html) that result in full table scans and consider adding [secondary indexes](../{{site.versions["stable"]}}/schema-design-indexes.html#create-a-secondary-index).
+
+### SQL Statement Contention 
+
+This graph shows the total number of SQL statements that experienced [contention](../{{site.versions["stable"]}}/transactions.html#transaction-contention) across the cluster.
```
Comment (ericharmeling) [inline cockroachcloud/metrics-page.md:75]:
```suggestion
This graph shows a moving average of the number of SQL statements that experienced [contention](../{{site.versions["stable"]}}/transactions.html#transaction-contention) across the cluster.
```

File: cockroachcloud/metrics-page.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+The statement contention metric is a counter that represents the number of statements that have experienced contention. If a statement experiences at least one contention "event" (i.e., the statement is forced to wait for another transaction), the counter is incremented at most once.
```
Comment (ericharmeling) [inline cockroachcloud/metrics-page.md:77]:
```suggestion

```

This paragraph seems a bit unnecessary.

**Review by ericharmeling — COMMENTED** [review]
Looks good! See my comments below.

After https://github.com/cockroachlabs/managed-service/pull/11892 is merged, the tooltips should be correct for all of these graphs.

**Review by rmloveland — APPROVED** [review]
LGTM, added some suggestions for more links

**Review by ericharmeling — APPROVED** [review]

**Pull request merged**